client/src/components/DataFiles/DataFilesModals/DataFilesPreviewModal.js

```diff
@@ -14,9 +14,16 @@ const DataFilesPreviewModal = () => {
   const dispatch = useDispatch();
   const isOpen = useSelector(state => state.files.modals.preview);
   const params = useSelector(state => state.files.modalProps.preview);
-  const previewHref = useSelector(state => state.files.preview.href);
-  const previewContent = useSelector(state => state.files.preview.content);
-  const isLoading = useSelector(state => state.files.preview.isLoading);
+  const {
+    href: previewHref,
+    content: previewTextContent,
+    error: previewError,
+    isLoading
+  } = useSelector(state => state.files.preview);
+  const hasError = previewError !== null;
+  const previewUsingTextContent =
+    !isLoading && !hasError && previewTextContent !== null;
```

Comment:
should this be content instead of previewTextContent?

Reply:
Just to make sure i'm following you.  Are you suggesting to change:
  ```
const {
    href: previewHref,
    content: previewTextContent,
    error: previewError,
    isLoading
  } = useSelector(state => state.files.preview);
```

be changed to something like:
```

  const {
    href,
    content,
    error,
    isLoading
  } = useSelector(state => state.files.preview);
```

If so, 👍 .  if not, can you explain some more.

Reply:
@jchuahtacc stopped renaming things in 4fc0bc9